tests/integration/targets/test_azure_manage_networking_stack/tasks/main.yaml

```
@@ -0,0 +1,161 @@
+---
+- name: Test role cloud.azure_ops.azure_manage_networking_stack with default options
+  block:
+    # Create Networking Stack
+    - name: Create Networking stack
+      ansible.builtin.include_role:
+        name: cloud.azure_ops.azure_manage_networking_stack
+      vars:
+        azure_manage_networking_stack_operation: create
```

Following my previous comment, `azure_manage_networking_stack_resource_group` var need to be indicated here:
```suggestion
        azure_manage_networking_stack_operation: create
        azure_manage_networking_stack_resource_group: "{{ resource_group }}"
```
(The same for all other `cloud.azure_ops.azure_manage_networking_stack` calls)